azure-pipelines.perf-test.yml

```diff
@@ -24,11 +24,14 @@ steps:
 
   # Fluent perf-test must run before Fabric perf-test until they are consolidated.
   - script: |
-      if [ $buildReason != "PullRequest" ]; then
-        yarn perf:test:base
-      else
-        yarn perf:test
-      fi
+      yarn perf:test:base
+    condition: ne(variables.buildReason, 'PullRequest')
```

Comment:
note for @ecraig12345 who asked before - since this job is triggered "manually" by a the main PR build pipeline, the original reason is lost. So this is necessary

Reply:
Yeah, that's why I put [this comment](https://github.com/OfficeDev/office-ui-fabric-react/blob/dd12706484118dc69defc2356d2426351314af3d/azure-pipelines.perf-test.yml#L19) in the file...

Reply:
Ah, the comment was conveniently outside the displayed diff region. 🙄